dsmr_exporter/dsmr_exporter.py

```diff
@@ -200,29 +200,30 @@ def telegram_to_json(self, telegram):
                     pass
                 doc[key] = value
```

Comment:
Note that this will store the `value` even if it failed to be parsed as a float. I don't know if this is intended behaviour or not?